lib/matplotlib/pyplot.py

```diff
@@ -2411,8 +2411,11 @@ def plotfile(fname, cols=(0,), plotfuncs=None,
 
     if plotfuncs is None:
         plotfuncs = dict()
-    r = mlab.csv2rec(fname, comments=comments, skiprows=skiprows,
-                     checkrows=checkrows, delimiter=delimiter, names=names)
+    from matplotlib.cbook import mplDeprecation
```

Comment:
Should this whole method be deprecated since it depends on `csv2rec`?

Reply:
Yes, and point people at pandas.

Reply:
You want to deprecate `plt.plotfile` for Pandas? Probably should be an independent PR.